src/effects/effects_base/internal/effectsprovider.cpp

```diff
@@ -108,14 +121,21 @@ bool EffectsProvider::loadEffect(const EffectId& effectId) const
         // If an effect is not a VST and is in m_effects, then it's a built-in effect and it's loaded already.
         return true;
     }
-    IF_ASSERT_FAILED(it->family == EffectFamily::VST3) {
-        LOGE() << "unknown family: " << static_cast<int>(it->family);
-        return false;
-    }
-    IF_ASSERT_FAILED(vstEffectsRepository()) {
-        return false;
+    switch (it->family) {
+        case EffectFamily::VST3:
+            IF_ASSERT_FAILED(vstEffectsRepository()) {
```

Comment:
`IF_ASSERT_FAILED(vstEffectsRepository())` wouldn't this and similar checks always be true, when the module is disabled it will be stubbed isn't it?